Verbotene Autoren/about.html

```diff
@@ -1,7 +1,10 @@
-<html>
+<!doctype html>
+<html lang="de">
+
     <head>
+        <meta charset="utf-8">
         <link rel="stylesheet" type="text/css" href="main.css">
-    </head>
+    </head>s
```

Comment:
did your CMD key not register for a CMD+s shortcut?

Reply:
Yes. My left cmd key sometimes does not work.